proto/headscale/v1/machine.proto

```diff
@@ -18,7 +18,7 @@ message Machine {
     string machine_key  = 2;
     string node_key     = 3;
     string disco_key    = 4;
-    string ip_address   = 5;
+    repeated string ip_addresses   = 5;
     string name         = 6;
```

Comment:
I do not have much experience with gRPC, but I *suppose* this might not be the best way to handle this change. Should I have allocated a new field number? I went with this because I was just trying to get it to work in the simplest possible way. Though even if a new number is allocated, the API will still not be compatible between versions. Do clients transmit enough metadata for things to fail without any strange misparsing if I just leave it like this?